mne/datasets/limo/limo.py

```diff
@@ -0,0 +1,231 @@
+        "~/mne_data" directory is used. If the LIMO dataset
+        is not found under the given path, the data
+        will be automatically downloaded to the specified folder.
+    force_update : bool
+        Force update of the dataset even if a local copy exists.
+    update_path : bool | None
+        If True, set the MNE_DATASETS_LIMO_PATH in mne-python
+        config to the given path. If None, the user is prompted.
+    %(verbose)s
+
+    Returns
+    -------
+    path : str
+        Local path to the given data file.
+
+    Notes
+    -----
+    For example, one could do:
+
+        >>> from mne.datasets import limo
+        >>> limo.data_path(subject=1, path=os.getenv('HOME') + '/datasets') # doctest:+SKIP
+
+    This would download the LIMO data file to the 'datasets' folder,
+    and prompt the user to save the 'datasets' path to the mne-python config,
+    if it isn't there already.
+
+    References
+    ----------
+    .. [1] Guillaume, Rousselet. (2016). LIMO EEG Dataset, [dataset].
+           University of Edinburgh, Centre for Clinical Brain Sciences.
+           https://doi.org/10.7488/ds/1556.
+    .. [2] https://datashare.is.ed.ac.uk/handle/10283/2189?show=full
+    """  # noqa: E501
+    # set destination path for download
+    key = 'MNE_DATASETS_LIMO_PATH'
+    name = 'LIMO'
+    path = _get_path(path, key, name)
+    limo_dir = op.join(path, 'MNE-limo-data')
+    subject_id = 'S%s' % subject
+    destination = op.join(limo_dir, '%s.zip') % subject_id
+
+    # url for subject in question
+    url = op.join(root_url, subject_ids[subject_id], '?zip=')
+
+    # check if LIMO directory exists; update if desired
+    if not op.isdir(limo_dir) or force_update:
+        if op.isdir(limo_dir):
+            shutil.rmtree(limo_dir)
+        if not op.isdir(limo_dir):
+            os.makedirs(limo_dir)
+
+    # check if subject in question exists
+    if not op.isdir(op.join(limo_dir, subject_id)):
+        os.makedirs(op.join(limo_dir, subject_id))
+        _fetch_file(url, destination, print_destination=False)
+
+        # check if download is a zip-folder
+        if any(group.endswith(".zip") for group in op.splitext(destination)):
+            if not op.isdir(op.join(limo_dir, subject_id)):
+                os.makedirs(op.join(limo_dir, subject_id))
+            with zipfile.ZipFile(destination) as z1:
+                files = [op.join(limo_dir, file) for file in z1.namelist()]
+                stdout.write('Decompressing %g files from\n'
+                             '"%s" ...' % (len(files), destination))
+                z1.extractall(op.join(limo_dir, subject_id))
+                stdout.write(' [done]\n')
+                z1.close()
+                os.remove(destination)
+
+    # update path if desired
+    _do_path_update(path, update_path, key, name)
+
+    return limo_dir
+
+
+@verbose
+def load_data(subject, path=None, interpolate=False, force_update=False,
```

Comment:
I would not put the interpolation logic in the loader

Reply:
Ok, I guess we can remove the interpolation and let people interpolate missing channels  (currently marked as "bad") later if desired. Would that be a better solution?

Reply:
At least we make no API commitment that way and don't lose much by pulling it out.

Reply:
Yes. Remove the option

Reply:
Ok. I removed the option in the loader and added a comment in ``plot_limo_data`` regarding interpolation of missing channels.